.github/ISSUE_TEMPLATE/new_entry.md

```diff
@@ -1,32 +1,23 @@
 ---
 name: New Entry
-about: Tell about a new app, service or website that you found out about the deletion process
-title: ''
+about: Tell us about a new service you've discovered, and how their deletion process works.
+title: 'Add (the service's Name).'
 labels: 'new_entry'
 assignees: ''
-
 ---
 
 ## Backstory
 
-<!-- Tell a bit about how you approached the deletion request and how it went -->
+<!-- Tell us in short about how you approached the deletion request and how it went. -->
 
 ## Data
 
-<!-- Fill in the fields below, delete any optional ones that do not apply or that you don't have the information -->
-
 - `name`: <!-- The name of the service -->
-- `url`: <!-- The url of the account-deletion page. If no such page exists, the url should be a contact or help page explaining the process of account deletion -->
-- `url_CODE`: <!-- *(optional)* Use the language `CODE` as suffix of the url field to provide language-specific deletion codes, shown on the respective language page -->
-- `difficulty`: <!-- one of the difficulties below -->
-<!--
-    - `easy`: Sites with a simple process such as a 'delete account' button
-    - `medium`: Sites that do allow account deletion but require you to perform additional steps
-    - `hard`: Sites that require you to contact customer services or those that don't allow automatic or easy account deletion
-    - `impossible`: For sites where it's basically impossible to totally delete your account, even if you contact them
--->
-- `notes`: <!-- *(optional)* Notes will be shown when someone hovers on that service, they include additional information you might need to delete your account -->
-- `email`: <!-- *(optional)* If you have to send an email to a company to cancel your account, add the email address here -->
-- `email_subject`: <!-- *(optional)* Set the subject for the email link. If unset, the default text is "Account Deletion Request" -->
-- `email_body`: <!-- *(optional)* Set the body for the email link. If unset, the default text is "Please delete my account, my username is XXXXXX" -->
-- `domains`: <!-- This is used by the [Chrome extension](https://github.com/jdm-contrib/justdelete.me-chrome-extension) -->
+- `url`: <!-- The url of the account-deletion page. If non exists, the url should be a contact or help page explaining the deletion process. -->
+- `url_CODE`: <!-- *(optional)* Use the language `CODE` as suffix of the url field to use language-specific deletion codes, shown on the respective language page -->
+- `difficulty`: <!-- Use one of the following difficulties easy/medium/hard/impossible, to learn more about all of the different difficulties [press here](https://github.com/jdm-contrib/justdelete.me-chrome-extension](https://github.com/jdm-contrib/jdm/blob/master/CONTRIBUTING.md). -->
```

Comment:
I don't think this hyperlink works in a markdown comment, did you test it? (even though we had one for the extension already)

Reply:
It works in markdown if i recall correctly.